akka-docs/rst/java/http/implications-of-streaming-http-entity.rst

```diff
@@ -25,7 +26,7 @@ Consuming the HTTP Response Entity (Client)
 
 The most common use-case of course is consuming the response entity, which can be done via
 running the underlying ``dataBytes`` Source. This is as simple as running the dataBytes source,
-(or on the server-side using directives such as 
+(or on the server-side using directives such as
```

Comment:
Oh this seems like dead ended sentence, could you add a link to `extractDataBytes`)?